ingest/src/main/java/com/microsoft/azure/kusto/ingest/AzureStorageClient.java

```diff
@@ -27,7 +27,7 @@
 import java.nio.file.Files;
 import java.util.zip.GZIPOutputStream;
 
-class AzureStorageClient {
+public class AzureStorageClient {
```

Comment:
If making this class public so it can be used, I suggest adding javadocs for the class and the API (`public`) methods.

Reply:
It's not really meant for the consumption of users, it's only public because it has to be used in a subpackage
Do you have another idea.

Reply:
Way out of scope, but one way I've done this is to have an api package with everything that's fit for public use being `public`, + a core package with classes like this.

Resolving :-)